tests/test-maximize.lua

```diff
@@ -38,8 +38,10 @@ for _, gravity in ipairs { "NORTH_WEST", "NORTH", "NORTH_EAST", "WEST",
         assert(not c.maximized_vertical  )
         assert(not c.maximized           )
         assert(not c.fullscreen          )
+        assert(not c.is_immobilized      )
```

Comment:
This assertion fails on Travis (I only checked one built, but I guess all of them).

Reply:
`c.is_immobilized` is a function and not a boolean when printing it.
Guess I will figure it out with @Elv13 later.

Reply:
Uhm, right. `is_` is one of the prefixes that is recognized just like `get_` and `set_`. So, the function needs to be called `is_is_immobilized`. But actually, stop using `is_immobilized` and call the property `immobilized` since the prefix `is_` seems unnecessary.